vxquery-core/src/test/java/org/apache/vxquery/datamodel/AbstractPointableTest.java

```diff
@@ -28,7 +28,9 @@
 
     protected void getTaggedValuePointable(Object value, IPointable result) throws IOException {
```

Comment:
It may be helpful to add an other argument for including the type tag. This way you can create strings without the type tag. "protected void getTaggedValuePointable(Object value, boolean includeTag, IPointable result) throws IOException "